web3/_utils/module_testing/eth_module.py

```diff
@@ -1096,6 +1096,27 @@ async def test_async_eth_syncing(self, async_w3: "Web3") -> None:
             assert is_integer(sync_dict['currentBlock'])
             assert is_integer(sync_dict['highestBlock'])
 
+    @pytest.mark.asyncio
+    async def test_async_eth_get_storage_at(
+        self, async_w3: "Web3", emitter_contract_address: ChecksumAddress
+    ) -> None:
+        storage = await async_w3.eth.get_storage_at(emitter_contract_address, 0) # type: ignore
+        assert isinstance(storage, HexBytes)
+
+    @pytest.mark.asyncio
+    async def test_async_eth_get_storage_at_ens_name(
```

Comment:
We don't have the middleware async-ified yet for ens names, so you can mark this one as an xfail:
```suggestion
    @pytest.mark.xfail
    async def test_async_eth_get_storage_at_ens_name(
```